models/hovernet/Dockerfile_hovernet

```diff
@@ -20,10 +9,11 @@ COPY . .
 ENV DEBIAN_FRONTEND=noninteractive
 
 # install libraries
-RUN apt-get update && apt-get install -y wget
+RUN apt-get clean && apt-get update
+RUN apt-get install -y wget
 RUN apt-get install -y git-all
 RUN apt install -y build-essential checkinstall libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev
-RUN apt update && apt install -y openslide-tools libncurses-dev libgdbm-dev libz-dev tk-dev libsqlite3-dev libreadline-dev liblzma-dev libffi-dev libssl-dev
```

Comment:
does it find all the libraries even without the update?

Reply:
Yes it should since there is an apt-update in the first line of this install block.